src/encoded/schemas/experiment_hi_c.json

```diff
@@ -58,7 +58,8 @@
                 "2% Formaldehyde",
                 "2.5% Formaldehyde",
                 "3.5% Formaldehyde",
-                "1% Formaldehyde and 3mM DSG"
+                "1% Formaldehyde and 3nM DSG",
```

Comment:
What happened to the 3mM DSG option? We have 45 experiments in the portal that use that.

Reply:
good catch!